index.template.html

```diff
@@ -59,7 +59,7 @@
                 }, {
                     name: "Matthias Kovatsch",
                     company:    "Huawei",
```

Comment:
We should add something like "Until DATETIME" to indicate when he stopped working there. Similar for Victor Charpenay.